application/feature/characters/core/api/src/main/java/com/stepango/blockme/feature/characters/core/api/domain/model/ICharacter.kt

```diff
@@ -1,6 +1,6 @@
-package com.stepango.blockme.feature.characters.list.api.domain.model
+package com.stepango.blockme.feature.characters.core.api.domain.model
 
-interface ICharacterItem {
+interface ICharacter {
```

Comment:
Good improvement!
Now we have only 3 types of "Character"!
But we can do more, what do you think, if we'll remove `CharacterItem` and `CharacterDetail`? I think about added single entity implementation of `ICharacter` into core:impl, for example 
`class Character : ICharacter`
And other domains `Character Details/Items/Favorites` will use only ICharacter type with default core implementation under hood.

In that case, will be only one `Character` single entity with unique structure placed in the core target.
What do you think?